packages/svg/src/icons/CloudSavedIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+import { IconProps } from "./types";
+
+export const CloudSavedIcon: FC<IconProps> = ({
+    width,
+    height,
+    ...props
```

Comment:
some icons are supporting additional props but others dont ,  i think every icon should support all the svg attributes .